Shared/Flows/Onboarding/CodeViewController.swift

```diff
@@ -12,7 +12,6 @@ import Parse
 import TMROLocalization
 import Combine
 
-#warning("Move main actor to the base view controller.")
```

Comment:
Making the BaseVC main actor still has a lot of problems in beta 5. I'm going to wait until non-beta Xcode 13 before I start tinkering with this stuff.